examples/alchemical_model/usage.sh

```diff
@@ -0,0 +1,35 @@
+#!\bin\bash
```

Comment:
We don't want this for each model in the docs, right @PicoCentauri? However, maybe we can leave the example here and run it from the test CI of the alchemical model

Reply:
yeah, I would not duplicate this file. If we want this to be a test, let's make it a test and not an example.

Reply:
Fine, then I'll delete this file